### DISCUSSION TITLE
请问这里的规则可以转换成geo文件吗？

### DISCUSSION THREAD

**Discussion by GeorgeRudd** [blackmatrix7/ios_rule_script#991] [Q&A]
这里的规则很全，但是xray只能用geo文件，请问这里的规则可以转换成geo文件吗，比如 geosite:chinamax geoip:cloudflare 之类

**Comment by drangjchen** [comment]
可以的，用Github Actions就可以定制。
参考配置：https://github.com/drangjchen/clash-geosite
